For each protocol $prot, which is moved into subpackage $subname,
is installed file /usr/share/dovecot/protocols.d/$subname.conf with:

    protocols = $protocols $prot

--- a/doc/example-config/dovecot.conf
+++ b/doc/example-config/dovecot.conf
@@ -20,8 +20,9 @@
 # options. The paths listed here are for configure --prefix=/usr
 # --sysconfdir=/etc --localstatedir=/var
 
-# Protocols we want to be serving.
-#protocols = imap pop3 lmtp submission
+# Enable installed protocols.
+#protocols = imap
+!include_try /usr/share/dovecot/protocols.d/*.conf
 
 # A comma separated list of IPs or hosts where to listen in for connections. 
 # "*" listens in all IPv4 interfaces, "::" listens in all IPv6 interfaces.
--- a/src/master/master-settings.c.orig
+++ b/src/master/master-settings.c
@@ -206,7 +206,7 @@
 	.state_dir = PKG_STATEDIR,
 	.libexec_dir = PKG_LIBEXECDIR,
 	.instance_name = PACKAGE,
-	.protocols = "imap pop3 lmtp",
+	.protocols = "imap",
 	.listen = "*, ::",
 	.ssl = "yes:no:required",
 	.default_internal_user = "dovecot",
